satip/app.py

```diff
@@ -141,37 +192,60 @@ def run(
         )
         # 1. Get data from API, download native files
         with tempfile.TemporaryDirectory() as tmpdir:
-            download_manager = EUMETSATDownloadManager(
-                user_key=api_key,
-                user_secret=api_secret,
-                data_dir=tmpdir,
-                native_file_dir=save_dir_native,
-            )
-            if cleanup:
-                log.debug("Running Data Tailor Cleanup", memory=utils.get_memory())
-                download_manager.cleanup_datatailor()
-                return
+
             start_date = pd.Timestamp(start_time, tz="UTC") - pd.Timedelta(history)
             log.info(
                 f"Fetching datasets for {start_date} - {start_time}", memory=utils.get_memory()
             )
-            datasets = download_manager.identify_available_datasets(
-                start_date=start_date.strftime("%Y-%m-%d-%H:%M:%S"),
-                end_date=pd.Timestamp(start_time, tz="UTC").strftime("%Y-%m-%d-%H:%M:%S"),
-            )
-            # Check if any RSS imagery is available, if not, fall back to 15 minutely data
-            if (len(datasets) == 0) or use_backup:
-                log.warn(
-                    f"No RSS Imagery available or using backup ({use_backup=}), "
-                    f"falling back to 15-minutely data",
+
+            if not use_iodc:
```

Comment:
Usually easier to follow the positive logic flow

Reply:
yea good point, ill try to change this